MockedRethinkDB.js

```diff
@@ -0,0 +1,334 @@
+const filter = require("lodash.filter");
+const map = require("lodash.map");
+const flatMap = require("lodash.flatmap");
+const rimraf = require("rmfr");
```

Comment:
I think `rimraf` is the more popular library to do this. `rmrf` looks like it only has ~400 downloads per month

Reply:
@rrdelaney Ooh, I've got it.  `rmrf` is it's own module.   However, `rmfr` (flipped "r" and "f") does just wrap `rimraf`.

I can take this in either direction.

Reply:
I'd say just use `rimraf` 🙂 It's super popular and there's low risk in it

Reply:
@rrdelaney So I should promisify raw `rimraf` instead of using `rmfr`'s already promisified module?

Reply:
I think you can use [`util.promisify`](https://nodejs.org/api/util.html#util_util_promisify_original) to do it for you